doc/old_and_new_proposal.md

```diff
@@ -660,6 +661,10 @@ following options.
     consequence, the interface presented to the user by default will be
     different (containing different questions) depending of the value of this
     property.
+  * `multidisk_first`
+    When set to `true`, the initial proposal will be performed over all
+    available candidate disks, which means that the proposal will try to use
+    several disks before jumping to adjust the settings.
```

Comment:
NP: not sure if this version is better understandable:

When set to `true`, the initial proposal considers all disks as possible destination for the volumes to allocate. If set to `false`, the proposal tries to use individual disks to allocate the volumes, and it only uses all disks together when the volumes cannot be allocated in any individual disk.

Reply:
Thanks!